src/main/java/io/github/fusionflux/portalcubed/content/door/LockingChamberDoorBlock.java

```diff
@@ -0,0 +1,108 @@
+package io.github.fusionflux.portalcubed.content.door;
+
+import io.github.fusionflux.portalcubed.content.PortalCubedSounds;
+import net.minecraft.core.BlockPos;
+import net.minecraft.server.level.ServerLevel;
+import net.minecraft.sounds.SoundEvent;
+import net.minecraft.sounds.SoundSource;
+import net.minecraft.util.RandomSource;
+import net.minecraft.util.StringRepresentable;
+import net.minecraft.world.InteractionHand;
+import net.minecraft.world.InteractionResult;
+import net.minecraft.world.entity.Entity;
+import net.minecraft.world.entity.player.Player;
+import net.minecraft.world.level.Level;
+import net.minecraft.world.level.block.Block;
+import net.minecraft.world.level.block.state.BlockState;
+import net.minecraft.world.level.block.state.StateDefinition;
+import net.minecraft.world.level.block.state.properties.DoubleBlockHalf;
+import net.minecraft.world.level.block.state.properties.EnumProperty;
+import net.minecraft.world.level.gameevent.GameEvent;
+import net.minecraft.world.phys.BlockHitResult;
+import org.jetbrains.annotations.Nullable;
+
+import java.util.Locale;
+import java.util.Optional;
+
+public class LockingChamberDoorBlock extends ChamberDoorBlock {
+	public static final EnumProperty<State> STATE = EnumProperty.create("state", State.class);
+	public static final int UNLOCK_DELAY = 5;
+
+	public LockingChamberDoorBlock(Properties settings) {
+		super(settings);
+	}
+
+	@Override
+	protected void createBlockStateDefinition(StateDefinition.Builder<Block, BlockState> builder) {
+		builder.add(HALF, FACING, STATE, HINGE, POWERED);
```

Comment:
you should call super so you don't need to specify all of them

Reply:
Can't in this case, because I have to replace the open property with the state property

Reply:
comment that then